features/mailinglist/put.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Test the UDB3 labels API
+
+  Background:
+    Given I am using the UDB3 base URL
+    And I am using an UiTID v1 API key of consumer "uitdatabank"
+    And I am authorized as JWT provider v1 user "centraal_beheerder"
+    And I send and accept "application/json"
+
+  Scenario: Subscribe to newsletter
```

Comment:
I am in doubt this needs a feature test, because (like we discussed last Friday) this pollutes a testing environment.  If we decide to keep, I would also include an unhappy path with an email like `mailformedmailadres@` or something like that.

Reply:
Added a tag, and changed the Makefile so it auto skips this test